pyproject.toml

```diff
@@ -27,14 +33,34 @@ Homepage = "https://libraryofcongress.github.io/bagit-python/"
 [tool.ruff]
 target-version = "py38"
 
-
-[tool.setuptools_scm]
-
 [tool.isort]
 line_length = 110
 default_section = "THIRDPARTY"
 known_first_party = "bagit"
 
+[tool.pytest.ini_options]
+testpaths = ["test.py"]
+addopts = "-v"
+
 [tool.coverage.run]
 branch = true
-include = "bagit.py"
+include = "src"
+
+[tool.hatch.version]
+source = "vcs"
+
+[tool.hatch.build.targets.sdist]
+packages = ["src/bagit"]
+include = [
+  "src/bagit/*.py",
+  "src/bagit/locale/*.po",
+  "src/bagit/locale/*.mo",
+]
+
+
```

Comment:
Adding these here means that ruff will ensure they are available when running tests.